### DISCUSSION TITLE
Can't start container after build image with Dockerfile

### DISCUSSION THREAD

**Discussion by hongfy197** [OvenMediaLabs/OvenMediaEngine#1117] [Help]
I built a docker image using the Dockerfile and can't run the image it show error like below

<img width="1373" alt="Screenshot 2023-02-24 at 9 48 34 AM" src="https://user-images.githubusercontent.com/40728555/221079781-086b241e-994f-4e2a-a983-fd6c16b164fd.png">

**Comment by command-tab** [comment]
What does your Docker run command look like?

**Comment by hongfy197** [comment]
docker run -d \
-p 1935:1935 \
-p 3333:3333 \
-p 3334:3334 \
-p 3478:3478 \
-p 9000:9000 \
-p 9999:9999/udp \
-p 4000:4000/udp \
--name ovenmediaengine \
-t ome-origin

**Comment by getroot** [comment]
Please provide all information requested by the template before posting a question. It can save both of us time.

First of all, it looks like you built the docker image using the master branch. Is that right? Because the master branch is a development version, it can behave incorrectly at any time. See if the issue reproduces in the latest release as well.
